modules/oadp-pod-crash-set-resource-request-restic.adoc

```diff
@@ -22,8 +22,9 @@ kind: DataProtectionApplication
 configuration:
   restic:
     podConfig:
-      resourceAllocations:
+      resourceAllocations: <1>
         requests:
-          cpu: 500m
-          memory: 256Mi
+          cpu: 1000m
+          memory: 16Gi
 ----
+<1> The `resourceAllocations` listed are for average usage.
```

Comment:
@weshayutin , @shubham-pampattiwar 
We tested the spec with limits & requests.
the PR includes 2 spec changes for "requests" without "limit"

Reply:
According to @shubham-pampattiwar we don't document the limits.

Reply:
+1 OADP does not set the limits on velero pod